src/components/cell-editor.jsx

```diff
@@ -112,9 +112,9 @@ class CellEditor extends React.Component {
       lineNumbers: true,
       keyMap: 'sublime',
       extraKeys: {
-        'Ctrl-Space': this.props.cellType === 'javascript' ? this.autoComplete : undefined,
+        'Ctrl-Space': this.props.codeMirrorName === 'javascript' ? this.autoComplete : undefined,
```

Comment:
Should this key off of the `languageId`?  I think that's probably the more iodide-specific and stable thing to use than `codeMirrorName`, which as @hamilton pointed out, may change if we change syntax highlighters.

Also, I think eventually (though not necessarily in this PR), we'll want to make the autocompletion replacable with a callback, too.

Reply:
Ah good point-- it actually does key off the `langauageId`, but down in the `mapStateToProps`.

+1 on replaceable autocomplete. that's definitely something we'll want to figure out.